pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap

```diff
@@ -152,7 +152,7 @@ exports[`SearchFilter tests should render default Search component 1`] = `
       <FormSelectOption
         isDisabled={false}
         key="Module type"
-        label="Module type"
+        label="Content type"
         required={false}
         value="Module type"
```

Comment:
Can we also consider changing other module instances changes within this PR?

Reply:
that would be a different effort. This ticket I'm working on is weighted as 0.5